src/stats.rs

```diff
@@ -88,9 +89,13 @@ struct MessageStats {
     only_to_self: u32,
 }
 
+/// Where a securejoin invite link or QR code came from.
+/// This is only used if the user enabled sending of statistics.
```

Comment:
"... enabled `StatsSending`". More convenient for grepping if one doesn't know the exact option name